### ISSUE TITLE
owncloud app not installable 

### ISSUE THREAD

**Issue by evandrofisico** [messagerie-melanie2/Roundcube-Plugin-ownCloud#3]
The owncloud app part is not installable, as owncloud does not show the app on owncloud 9.0. Is there some hidden mechanism for forcing installation or is it necessary to enable development mode?

**Comment by tpayen** [comment]
Hm, normally you just have to put the 'roundcube_external' in the owncloud apps folder. And enable it in ownCloud Apps menu. I don't think you need to enable development mode

**Comment by mckaygerhard** [comment]
for install u must doit manually @evandrofisico using console commands in bash linux box, its not a win click-next-click-next

as easy as mention only put hole directoy in the apps directory and then go to apps panel.. but carefully, since oc 8.2 there's 2 modes of install apps, user mode and admin mode.. when u do as is, and user is admin, will enable ofr all users, and recomends for untested plugins like that does not filter for groups and let enalbed for all

**Comment by evandrofisico** [comment]
Yes, I tried the usual installation mode of copying roundcube_external folder into the apps folder, but no sign of it in the apps panel, just like I used to do on oc7, but apparently it is some issue with the way oc 9.x register apps.

**Comment by tpayen** [comment]
Maybe you need to change the rights of the folder by using chmod or/and chown (there is some issues with zip files)

**Comment by mckaygerhard** [comment]
@evandrofisico please use instruction of pull request in #6 

this issue are solved trhought #6
